xml/dm_registers.xml

```diff
@@ -17,17 +17,16 @@
         time.  Eg. in a multi-hart system some might temporarily be powered
         down, or a system might support hot-swapping harts.
 
-        <field name="0" bits="31:23" access="R" reset="0" />
-        <field name="impebreak" bits="22" access="R" reset="Preset">
+        <field name="0" bits="31:24" access="R" reset="0" />
+        <field name="impebreak" bits="23" access="R" reset="Preset">
             If 1, then there is an implicit {\tt ebreak} instruction at the
             non-existent word immediately after the Program Buffer. This saves
             the debugger from having to write the {\tt ebreak} itself, and
             allows the Program Buffer to be one word smaller.
 
             This must be 1 when \Fprogbufsize is 1.
         </field>
-        <field name="0" bits="21" access="R" reset="0" />
-        <field name="dmerr" bits="20:18" access="R/W1C" reset="0">
+        <field name="dmerr" bits="22:20" access="R/W1C" reset="0">
```

Comment:
I wouldn't move this, unless you know it hasn't been implemented anywhere yet.
(FWIW, OpenOCD doesn't check these bits.)

Reply:
I know of nowhere that it has been implemented, wasn't even aware of its addition... We can discuss in meeting tomorrow

Reply:
I don't care about dmerr. It'd be nice if impebreak doesn't move (or if it does that we're certain about its final location with this change), because the small program buffer OpenOCD pull request uses it.